recipes/libcarna/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "libcarna" %}
+{% set version = "3.3.3" %}
+{% set repo_url = "https://github.com/kostrykin/Carna" %}
+{% set docs_url = "https://kostrykin.github.io/Carna/html" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: "{{ repo_url }}/archive/refs/tags/{{ version }}.zip"
+  sha256: 0ec99c1188c6ec81747e97a018a88eacad39a39c83c3b79fb669cd89cb9c7637
+
+build:
+  number: 0
+  skip: true # [not linux]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin="x.x") }}
```

Comment:
I think `name` should be really libcarna here.

Reply:
Maybe I'm misunderstanding something, but I think `name` is already defined as `libcarna` here:
https://github.com/kostrykin/bioconda-recipes/blob/074fc915509ca3b9d21c161c5085e9bf2f52fb22/recipes/libcarna/meta.yaml#L1